tests/ZeroMQMessageProducerTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+namespace Prooph\ServiceBusTest;
```

Comment:
This is also wrong :)
Am 24.09.2015 19:23 schrieb "Bradley Weston" <notifications@github.com>:

> In tests/ZeroMQMessageProducerTest.php
> <https://github.com/prooph/psb-zeromq-producer/pull/1#discussion_r40353602>
> :
>
> > @@ -0,0 +1,62 @@
> > +<?php
> > +
> > +namespace Prooph\ServiceBusTest;
>
> I thought that too but take a look at this one;
> https://github.com/prooph/psb-http-producer/blob/master/tests/HttpMessageProducerTest.php
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/prooph/psb-zeromq-producer/pull/1/files#r40353602>.
>